scout/server/blueprints/alignviewers/views.py

```diff
@@ -91,7 +97,18 @@ def sashimi_igv(institute_id, case_name, variant_id):
     )  # This function takes care of checking if user is authorized to see resource
 
     display_obj = controllers.make_sashimi_tracks(case_obj, variant_id)
-    return render_template("alignviewers/igv_sashimi_viewer.html", **display_obj)
+    controllers.set_session_tracks(display_obj)
+
+    response = Response(render_template("alignviewers/igv_sashimi_viewer.html", **display_obj))
+
+    @response.call_on_close
```

Comment:
This executes when the response is closed removes the session item

Reply:
Cool thing!